src/work-allocation-2/containers/task-assignment/task-assignment-container.component.ts

```diff
@@ -15,46 +16,49 @@ import { getAssigneeName, handleFatalErrors } from '../../utils';
 export const NAME_ERROR: ErrorMessage = {
   title: 'There is a problem',
   description: 'You must select a name',
-  fieldId: 'task_assignment_caseworker'
+  fieldId: 'case-worker-search__input'
```

Comment:
There's a disparity between this and the id given to the input in the HTML, which is `'task_assignment_caseworker'`.